docs/reference/slm/getting-started-slm.asciidoc

```diff
@@ -62,66 +77,56 @@ PUT /_slm/policy/nightly-snapshots
+retaining at least 5 and no more than 50 snapshots regardless of age. 
+
+You can specify additional snapshot configuration options to customize how snapshots are taken.
+For example, you could configure the policy to fail the snapshot 
+if one of the specified indices is missing. 
+For more information about snapshot options, see <<snapshots-take-snapshot,snapshot requests>>.
 
 [float]
 [[slm-gs-test-policy]]
 ==== Test the snapshot policy
 
-While snapshots taken by SLM policies can be viewed through the standard snapshot
-API, SLM also keeps track of policy successes and failures in ways that are a bit
-easier to use to make sure the policy is working.  Once a policy has executed at
-least once, when you view the policy using the <<slm-api-get-policy>>, 
-some metadata will be returned indicating whether the snapshot was sucessfully 
-initiated or not.
+A snapshot taken by {slm-init} is just like any other snapshot. 
+You can view information about snapshots in {kib} or get info with the snapshot APIs. 
+In addition, {slm-init } keeps track of policy successes and failures so you 
+have insight into how the policy is working. If the policy has executed at
+least once, the <<slm-api-get-policy>> API returns additional metadata
+that shows if the snapshot succeeded.
+
+You can manually execute a snapshot policy to take a snapshot immediately. 
+This is useful for taking snapshots before making a configuration change, 
+upgrading, or to test a new policy. 
+Manually executing a policy does not affect its configured schedule. 
 
-Instead of waiting for our policy to run, let's tell SLM to take a snapshot
-as using the configuration from our policy right now instead of waiting for
-1:30AM.
+For example, the following request manually triggers the `nightly-snapshots` policy:
 
 [source,console]
 --------------------------------------------------
 POST /_slm/policy/nightly-snapshots/_execute
 --------------------------------------------------
 // TEST[skip:we can't easily handle snapshots from docs tests]
 
-This request will kick off a snapshot for our policy right now, regardless of
-the schedule in the policy. This is useful for taking snapshots before making 
-a configuration change, upgrading, or for our purposes, making sure our policy
-is going to work successfully. The policy will continue to run on its configured
-schedule after this execution of the policy.
+
+After forcing the `nightly-snapshots` policy to run, 
+you will see the sucess or failure information if you retrieve the policy.
 
 [source,console]
 --------------------------------------------------
 GET /_slm/policy/nightly-snapshots?human
 --------------------------------------------------
 // TEST[continued]
 
-This request will return a response that includes the policy, as well as
-information about the last time the policy succeeded and failed, as well as the
-next time the policy will be executed.
+The response also shows when the policy is scheduled to execute next:
```

Comment:
I'd incorporate this with the above para. Kinda odd to have the info split over two paras with a snippet between.